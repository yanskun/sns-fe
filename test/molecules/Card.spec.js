import { mount } from '@vue/test-utils'
import Component from '~/components/molecules/Card'

describe('Card', () => {
  test('snapshot', () => {
    const wrapper = mount(Component)
    expect(wrapper.element).toMatchSnapshot()
  })
})
