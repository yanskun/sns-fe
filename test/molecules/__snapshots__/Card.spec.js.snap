// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card snapshot 1`] = `
<div
  class="card-wrapper v-card v-sheet theme--light"
/>
`;
